atlp-rwanda/atlp-devpulse-fn
# Admin view trainee applicants

### PR Description
As an admin, I want to be able to view the application cycle status. 
This includes a table with a list of all applicants in a cycle and the 
stage at which they are.

### Description of tasks that were expected to be completed

- View all applicants listed with their cycle, stage, and status.
- Admin can move the applicant to the next stage

###How has this been tested?

- Checkout to our branch ft-admin-view-applicants
- Install dependencies: `npm install` Run the app: `npm run dev`
- Log in as an ` admin` or `superAdmin` and you will be redirected to the dashboard page
- In the admins panel navigate to [Applicant-Stages](http://localhost:3000/#/admin/Applicant-Stages).
- Then you are able to see and change ap[applicants' details such as their stage
### Credentials


### Track PR (issue number & link)
Issue: [213](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/213)

![Screenshot 2024-10-25 at 13 46 13](https://github.com/user-attachments/assets/f600d36a-7143-4195-bcfb-49007493a9af)
![Screenshot 2024-10-25 at 14 01 21](https://github.com/user-attachments/assets/c0261794-23fc-4693-80b1-c9f57eec5d1e)
![Screenshot 2024-10-25 at 14 01 42](https://github.com/user-attachments/assets/89abe212-e96e-4b53-8e00-dd2cba7d795a)
